content/attacks/posts/2021-02-13-Alpha-Finance.md

```diff
@@ -0,0 +1,47 @@
+---
+date: 2021-02-13
+categories:
+    - 51%
+    - Protocol Exploit
+title: "Alpha Finance DeFi Hack: $37.5 Million Exploited"
+---
+
+
+## Summary
+
+On February 13, 2021, Alpha Finance, a DeFi project, experienced a hack resulting in a loss of $37.5 million. Later, Iron Bank (IB) unilaterally modified the smart contract configuration, preventing lenders on Alpha Homora (AH) from withdrawing their liquidity due to an unresolved debt. Despite negotiations and proposed solutions, IB maintained the freeze on funds, leading depositors to cease negotiations, receive goodwill funds from AH, and pursue legal action against IB.
+
+## Attackers
+
+The identity of the attackers remains unknown. The attack was performed using the [address](https://twitter.com/josebaredes/status/1360476183373242370) [0x905315602Ed9a854e325F692FF82F58799BEaB57](https://etherscan.io/address/0x905315602ed9a854e325f692ff82f58799beab57).
+
+ ## Losses
+
+The Alpha Finance DeFi hack resulted in [financial losses](https://cryptobriefing.com/alpha-finance-suffers-37-5-million-loss-major-attack/), with $37.5 million extracted from the project. The stolen funds were distributed among various destinations as follows:
+
+  + Iron Bank: 1,000 ETH
+  + Alpha Homora: 1,000 ETH
+  + Tornado.cash:  320 ETH 
+  + Attacker's wallet: 10,925 ETH (worth roughly $20 million)
+
+## Timeline
+
+- **February 13, 2021:** 
+In the initial stage, the attacker borrowed [1,000e^(18)](https://www.quadrigainitiative.com/casestudy/alphahomorahack.php) sUSD from HomoraBankv2, utilizing [UNI-WETH LP](https://www.halborn.com/blog/post/explained-the-alpha-homora-defi-hack-feb-2021) as collateral. During repayment, the attacker exploited a [rounding error](https://www.halborn.com/blog/post/explained-the-alpha-homora-defi-hack-feb-2021) in the protocol, paying slightly less than the owed amount. This manipulation resulted in a debt of 19709787742197 and a total borrow share of 1. Utilizing the accumulated debt, the attacker engaged in multiple transactions involving borrowing and lending sUSD between Alpha Homora and Iron Bank (Cream). This process enabled the attacker to amass a substantial hoard of cySUSD, which is a collateralized version of sUSD. Using the hoarded cySUSD, the attacker obtained loans of various assets, including 13.2k WETH, 3.6M USDC, 5.6M USDT, and 4.2M DAI. The USDC, USDT, and DAI were subsequently deposited into Aave to acquire aTokens, which were then added to Curve's a3Crv pool. The extracted Ether was distributed among different destinations, including Iron Bank, Alpha Homora, Tornado.cash, and the attacker's wallet.

```

Comment:
Please, leave only one or two short sentences in timeline, and move important details to the Summary section, as said in previous comments